src/Symfony/Component/ExpressionLanguage/ExpressionFunction.php

```diff
@@ -62,4 +62,43 @@ public function getEvaluator()
     {
         return $this->evaluator;
     }
+
+    /**
+     * Creates an ExpressionFunction from a PHP function name.
+     *
+     * @param string      $phpFunctionName        The PHP function name
+     * @param string|null $expressionFunctionName The expression function name (default: same than the PHP function name)
+     *
+     * @return self
+     *
+     * @throws \InvalidArgumentException if given PHP function name does not exist
+     * @throws \InvalidArgumentException if given PHP function name is in namespace
+     *                                   and expression function name is not defined
+     */
+    public static function fromPhp($phpFunctionName, $expressionFunctionName = null)
+    {
+        if (!function_exists($phpFunctionName)) {
+            throw new \InvalidArgumentException(sprintf('PHP function "%s" does not exist.', $phpFunctionName));
+        }
+
+        $reflection = new \ReflectionFunction($phpFunctionName);
+        if (!$expressionFunctionName && $reflection->inNamespace()) {
+            throw new \InvalidArgumentException(sprintf('An expression function name must be defined if PHP function "%s" is in namespace.', $phpFunctionName));
```

Comment:
`An expression function name must be defined when PHP function "%s" is namespaced.`?